dll/win32/kernel32/kernel32_vista/runonce.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * PROJECT:         ReactOS system libraries
```

Comment:
Thanks for the contribution!
Please, format the header according to our [Coding Style](https://reactos.org/wiki/Coding_Style) and this is good to go.

Reply:
committed